src/components/App.styles.ts

```diff
@@ -0,0 +1,11 @@
+import { createUseStyles } from "react-jss";
+
+export default createUseStyles({
+  container: {
+    display: "flex",
+    maxWidth: "1600px",
+    margin: "0 auto",
+    padding: "0.5rem",
+    width: "100%",
+  },
+});
```

Comment:
This could've just gone on "src/themes/default.ts", on the global main section, since is part of the main layout

Reply:
I'll move that to src/themes/default.ts global